tutorandroid/plugin.py

```diff
@@ -71,10 +109,10 @@
 )
 # Load patches from files
 for path in glob(
-    os.path.join(
-        pkg_resources.resource_filename("tutorandroid", "patches"),
-        "*",
-    )
+        os.path.join(
+            pkg_resources.resource_filename("tutorandroid", "patches"),
+            "*",
+        )
```

Comment:
Why the extra spaces? You should be able to get rid of them by running `black` on your file:

    black tutorandroid/plugin.py

Reply:
No idea how they got there! 🤔